packages/main/src/plugin/container-registry.ts

```diff
@@ -1132,7 +1132,7 @@ export class ContainerProviderRegistry {
     try {
       const authconfig = this.imageRegistry.getAuthconfigForImage(imageName);
       const matchingEngine = this.getMatchingEngineFromConnection(providerContainerConnectionInfo);
-      const pullStream = await matchingEngine.pull(imageName, {
+      const pullStream: NodeJS.ReadableStream = await matchingEngine.pull(imageName, {
```

Comment:
can't we omit NodeJS.ReadableStream there ?

Reply:
yes